ansible/roles/collectory/vars/main.yml

```diff
@@ -1,17 +1,21 @@
 # application versions
 # assets
 # collectory_version: "1.5.1"
-version: "{{ collectory_version | default('LATEST') }}"
+collectory_service_hostname: "{{ collectory_app | default('collectory') }}"
+
+collectory_service_system_user: "collectory"
 
 artifactId: "{{ collectory_app | default('generic-collectory') }}"
 
 groupId: "{{ collectory_group_id | default('au.org.ala') }}"
 # collectory does not need a value for classifier
 classifier: ''
 
-packaging: "war"
+packaging: "{{ collectory_packaging | default('war') }}"
+
+version: "{{ collectory_version | default('1.5.1') }}"
```

Comment:
Revert to the previous default version `LATEST`.

Reply:
done